packages/auth/src/authClients/clerk.ts

```diff
@@ -30,16 +31,16 @@ export const clerk = (client: Clerk): AuthClientClerk => {
     client,
     login: async (options?: SignInProps) =>
       clerkClient(client)?.openSignIn(options || {}),
-    logout: async () => new Promise((res) => clerkClient(client)?.signOut(res)),
```

Comment:
I do not know why this was formulated this way, but seemed superficial. If there is any uknown issue with this let me know.

Reply:
Agreed. Your update here makes sense.